.drone.star

```diff
@@ -1328,22 +1328,17 @@ def ocisServer(storage):
       'detach': True,
       'environment' : {
         #'OCIS_LOG_LEVEL': 'debug',
-        'STORAGE_STORAGE_HOME_DRIVER': '%s' % (storage),
-        'STORAGE_STORAGE_HOME_DATA_DRIVER': '%s' % (storage),
-        'STORAGE_STORAGE_OC_DRIVER': '%s' % (storage),
-        'STORAGE_STORAGE_OC_DATA_DRIVER': '%s' % (storage),
-        'STORAGE_STORAGE_HOME_DATA_TEMP_FOLDER': '/srv/app/tmp/',
-        'STORAGE_STORAGE_OCIS_ROOT': '/srv/app/tmp/ocis/storage/users',
-        'STORAGE_STORAGE_LOCAL_ROOT': '/srv/app/tmp/ocis/local/root',
-        'STORAGE_STORAGE_OWNCLOUD_DATADIR': '/srv/app/tmp/ocis/owncloud/data',
-        'STORAGE_STORAGE_OC_DATA_TEMP_FOLDER': '/srv/app/tmp/',
-        'STORAGE_STORAGE_OWNCLOUD_REDIS_ADDR': 'redis:6379',
+        'STORAGE_HOME_DRIVER': '%s' % (storage),
+        'STORAGE_OCIS_ROOT': '/srv/app/tmp/ocis/storage/users',
+        'STORAGE_LOCAL_ROOT': '/srv/app/tmp/ocis/local/root',
+        'STORAGE_OWNCLOUD_DATADIR': '/srv/app/tmp/ocis/owncloud/data',
+        'STORAGE_OWNCLOUD_REDIS_ADDR': 'redis:6379',
         'STORAGE_LDAP_IDP': 'https://ocis-server:9200',
         'STORAGE_OIDC_ISSUER': 'https://ocis-server:9200',
         'PROXY_OIDC_ISSUER': 'https://ocis-server:9200',
-        'STORAGE_STORAGE_OC_DATA_SERVER_URL': 'http://ocis-server:9164/data',
-        'STORAGE_DATAGATEWAY_URL': 'https://ocis-server:9200/data',
-        'STORAGE_FRONTEND_URL': 'https://ocis-server:9200',
+        'STORAGE_HOME_DATA_SERVER_URL': 'https://ocis-server:9200/data',
```

Comment:
why the change to :9200 ?

Reply:
The default is pointing to `http://localhost:9155/" but that didn't work. Threw an error about http responses to an https client.
So I thought maybe I have to put the proxy port there. But now I'm trying the old `OC_DATA_SERVER_URL`.